app/src/main/java/org/simple/clinic/summary/teleconsultation/contactdoctor/ContactDoctorSheet_New.kt

```diff
@@ -0,0 +1,13 @@
+package org.simple.clinic.summary.teleconsultation.contactdoctor
+
+import android.os.Bundle
+import org.simple.clinic.R
+import org.simple.clinic.widgets.BottomSheetActivity
+
+class ContactDoctorSheet_New : BottomSheetActivity() {
```

Comment:
Do we specifically need the `_New` prefix here? We can drop this since we will be deleting the older screen later, right?